### PR TITLE
Feature/our cv homepage

### PR DESCRIPTION
**Pull request by augusto-gs**
Created our-cvs.html and added Augusto´s CV to the file

### CODE REVIEW

File: our-cvs.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (BertaGM) [inline our-cvs.html:2]: Change language atribute

File: our-cvs.html [hunk 2]
```diff
@@ -21,6 +21,42 @@ <h1>Nuestra pagina web</h1>
...
+    <h2>Experiencia de trabajo</h2>
+    <article>
+      <h3>Auerolightning, Diseñador</h3>
+      <p>Aunando intereses, experiencias y conocimientos damos la mejor solucion como una agencia de Diseño en todas
+        sus
+        facetas para conseguir una imagen global, fuerte y consolidada</p>
+    </article>
+    <article>
+      <h3>Sim Sum, Interiorista</h3>
+      <h4>Interiorista</h4>
```
Comment (BertaGM) [inline our-cvs.html:34]: Delete h4 atribute, please

File: our-cvs.html (hunk 1 above)
Comment (augusto-gs) [reply our-cvs.html]: Done. Thanks for the feedback and attention to detail

File: our-cvs.html (hunk 2 above)
Comment (augusto-gs) [reply our-cvs.html]: Done. Thanks!👍

**Pull request merged**